tests/test_assemblies.py

```diff
@@ -481,4 +485,64 @@ def test_fields_present_abnormal_sets(self, identifier, field):
     ])
     def test_fields_present_cue_conflict(self, identifier, field):
         assembly = brainscore.get_assembly(f"brendel.Geirhos2021_{identifier}")
-        assert hasattr(assembly, field)
\ No newline at end of file
+        assert hasattr(assembly, field)
+
+@pytest.mark.private_access
+class TestBaker2022:
+
+    # tests alignments that are the same across normal and inverted assemblies
+    @pytest.mark.parametrize('identifier, length', [
+        ('normal', 3702),
+        ('inverted', 4320),
+    ])
+    def test_stimulus_set_assembly_alignment(self, identifier, length):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert assembly.stimulus_set is not None
+        assert assembly.stimulus_set.identifier == f'Baker2022_{identifier}_distortion'
+        assert set(assembly.stimulus_set["animal"]) == set(assembly["truth"].values)
+        assert set(assembly.stimulus_set["stimulus_id"]) == set(assembly["stimulus_id"].values)
+        assert len(assembly.presentation) == length
+
+    # tests counts that are the same across normal and inverted assemblies
+    @pytest.mark.parametrize('identifier', [
+        'normal',
+        'inverted',
+    ])
+    def test_same_counts(self, identifier):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert len({x for x in set((assembly["truth"]).values) if x==x}) == 9
```

Comment:
What is this weird `{x for x in ... if x==x}`?
```suggestion
        assert len(set((assembly["truth"]).values)) == 9
```